src/Microsoft.ML.Data/TrainCatalog.cs

```diff
@@ -151,7 +151,12 @@ private void EnsureStratificationColumn(ref IDataView data, ref string stratific
                     // Generate a new column with the hashed stratification column.
                     while (data.Schema.TryGetColumnIndex(stratificationColumn, out tmp))
                         stratificationColumn = string.Format("{0}_{1:000}", origStratCol, ++inc);
-                    data = new HashingEstimator(Host, origStratCol, stratificationColumn, 30).Fit(data).Transform(data);
+                    HashingTransformer.ColumnInfo columnInfo;
+                    if (seed.HasValue)
+                        columnInfo = new HashingTransformer.ColumnInfo(origStratCol, stratificationColumn, 30, seed.Value);
```

Comment:
I wish there was a way to use `??` but I don't see it.

Reply:
I could make seed nullable, and apply default value if it's null, but that would remove visibility for user regarding what number we use by default.

---
In reply to: [251147195](https://github.com/dotnet/machinelearning/pull/2241#discussion_r251147195) [](ancestors = 251147195)